mParticle-UrbanAirship.podspec

```
@@ -15,7 +15,8 @@ Pod::Spec.new do |s|
 
     s.ios.deployment_target = "14.0"
     s.ios.source_files      = 'mParticle-UrbanAirship/*.{h,m,mm}'
+    s.ios.resource_bundles = { 'mParticle-Localytics-Privacy' => ['mParticle-Localytics/PrivacyInfo.xcprivacy'] }
```

```suggestion
    s.ios.resource_bundles = { 'mParticle-UrbanAirship-Privacy' => ['mParticle-UrbanAirship/PrivacyInfo.xcprivacy'] }
```

Looks like this was copy pasted from Localytics without the package name and path updated.